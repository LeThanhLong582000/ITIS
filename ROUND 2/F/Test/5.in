100
425045 104512 565452
862182 659906 450998
617259 735959 776182
542594 349445 528110
27407 313815 431201
404229 911089 715620
30406 278238 71941
426454 749159 716363
662273 459402 463205
70214 853191 752450
690440 400737 123714
824659 668634 320178
489952 867808 973812
614690 829340 744145
930229 655545 669267
794453 16936 686181
847626 949353 474378
719747 569520 779398
748738 548123 601265
25982 454629 976623
368966 681078 690370
94633 158273 264389
372632 14094 999305
799025 647621 801756
514349 257618 425932
806101 658450 164630
397261 420634 620454
593355 295621 263311
527531 655092 553767
583852 780820 644198
502460 609958 4027
959548 19509 88927
780375 278316 839408
306989 75553 66935
669829 250634 401397
527576 532338 144175
564779 107610 138458
440398 191441 982794
661331 677641 394495
492032 561810 202023
292656 508135 973213
819564 11643 354233
849926 349145 662571
668875 236859 622877
175146 768088 597696
597115 195455 165073
817031 498847 378187
591702 996992 748022
884084 766361 65049
58092 222167 420744
486923 858488 977943
359036 715617 50207
983839 597959 875731
593333 903893 550474
721751 693351 884006
49416 20371 714030
783632 398350 631518
751492 725832 317734
155139 879436 386576
659801 201352 148610
106349 737256 157318
637597 227747 823636
915478 185216 107343
789051 583529 982008
469068 931314 601247
662092 382751 471833
723042 121095 335031
471843 486353 655275
533072 496372 856386
812504 973274 934844
146445 101196 201277
556688 107158 48663
463555 903808 908924
996554 283995 370042
191127 382398 936616
48372 869225 875742
255469 320662 237062
741489 154956 61801
468898 425643 622162
962741 992387 490159
134766 783442 973810
898539 851890 642876
394022 488475 177130
478649 572058 971837
392255 71720 550998
504702 709521 719017
56249 279330 248550
559005 163381 470626
474269 452158 86636
538476 915522 24985
639442 428662 909729
582987 836189 542739
146798 981308 540867
627690 758248 387478
734644 104619 114208
978129 548350 359126
582684 665740 785672
545932 127536 622591
865854 926166 388146
947447 988793 638382
