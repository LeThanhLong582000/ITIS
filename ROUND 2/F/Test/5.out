442993
326637
328741
360630
72397
104380
0
196829
333038
0
123715
238102
297978
264748
471976
686182
186326
464813
400940
273988
189129
100375
678922
476580
341332
156141
298541
263312
213103
223615
0
88928
670734
66936
401398
69707
138459
615876
189093
66123
378867
354234
581676
527447
2377
165074
348186
171366
65050
128335
303189
0
630806
119957
456203
371538
508400
171697
0
148611
0
616743
107344
593765
69501
375587
335032
320383
446543
387037
123263
48664
234336
370043
372673
27445
85935
61802
332709
230257
162567
344763
41339
439214
435767
257099
12735
433125
54374
0
560255
144769
0
128460
114209
359127
351308
520494
163917
298518
